CorsixTH/Lua/game_ui.lua

```diff
@@ -211,6 +211,10 @@ local scroll_keys = {
   right = {x =  10, y =   0},
   down  = {x =   0, y =  10},
   left  = {x = -10, y =   0},
+  ["keypad 8"] = {x =   0, y = -10},
```

Comment:
Is this still necessary? Looks to me like it would be fixed by the key remap in ui.lua